components/camel-cdi/pom.xml

```diff
@@ -190,113 +190,11 @@
 
     <profiles>
         <profile>
-            <id>weld-1.0</id>
+            <id>weld-2.0</id>
```

Comment:
Maybe call it weld3 since its weld 3 we use ;)

Reply:
Done